crates/bitwarden/src/mobile/vault/client_sends.rs

```diff
@@ -52,6 +52,35 @@ impl<'a> ClientSends<'a> {
 
         enc.decrypt_bytes(&buf, &None)
     }
+
+    pub async fn encrypt(&self, send_view: SendView) -> Result<Send> {
+        let enc = self.client.get_encryption_settings()?;
+
+        let send = send_view.encrypt(enc, &None)?;
+
+        Ok(send)
+    }
+
+    pub async fn encrypt_file(
+        &self,
+        send: Send,
+        decrypted_file_path: &Path,
+        encrypted_file_path: &Path,
+    ) -> Result<()> {
+        let data = std::fs::read(decrypted_file_path).unwrap();
+        let encrypted = self.encrypt_buffer(send, &data).await?;
+        std::fs::write(encrypted_file_path, encrypted)?;
+        Ok(())
+    }
+
+    pub async fn encrypt_buffer(&self, send: Send, buffer: &[u8]) -> Result<Vec<u8>> {
+        let enc = self.client.get_encryption_settings()?;
+        let key = Send::get_key(&send.key, enc, &None)?;
+        let enc = EncryptionSettings::new_single_key(key);
```

Comment:
Should `Send::get_key` return a `EncryptionSettings::new_single_key` immediately?

Reply:
I think the reason to have the `get_key` not return `EncryptionSettings` directly was to make it more testable, but that doesn't mean we can't have a helper function like this, because we do it in a couple of places: 
```rust
    pub(crate) fn get_encryption(
        key: &EncString,
        enc: &EncryptionSettings,
        org_id: &Option<Uuid>,
    ) -> Result<EncryptionSettings> {
        let key = Send::get_key(key, enc, org_id)?;
        Ok(EncryptionSettings::new_single_key(key))
    }
```